src/viam/app/billing_client.py

```diff
@@ -60,11 +60,12 @@ async def get_invoice_pdf(self, invoice_id: str, org_id: str, dest: str, timeout
             org_id (str): the ID of the org to request data from
             dest (str): filepath to save the invoice to
         """
-        request = GetInvoicePdfRequest(id=invoice_id, org_id=org_id)
-        response: GetInvoicePdfResponse = await self._billing_client.GetInvoicePdf(request, metadata=self._metadata, timeout=timeout)
-        data: bytes = response[0].chunk
-        with open(dest, "wb") as file:
-            file.write(data)
+        stream: Stream[GetInvoicePdfRequest, GetInvoicePdfResponse]
+        async with self._billing_client.GetInvoicePdf.open(timeout=timeout, metadata=self._metadata) as stream:
```

Comment:
type checking surfaced this issue, where we weren't streaming the response properly